pkgs/applications/version-management/rcshist/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib
+, stdenv
+, fetchurl
+}:
+stdenv.mkDerivation
```

Comment:
We don't use this style in packages, please format with nixpkgs-fmt or Alejandra

Reply:
This is output of `alejandra` couple months ago. Today it does following:

```
{
  lib,
  stdenv,
  fetchurl,
}:
stdenv.mkDerivation
{
  pname = "rcshist";
  version = "1.04";
  src = fetchurl {
    url = "https://invisible-island.net/datafiles/release/rcshist.tar.gz";
    sha256 = "01ab3xwgm934lxr8bm758am3vxwx4hxx7cc9prbgqj5nh30vdg1n";
  };

  meta = {
    description = "Utitity to display complete revision history of a set of RCS files";
    homepage = "https://invisible-island.net/rcshist/rcshist.html";
    license = lib.licenses.bsd2;
    maintainers = [lib.maintainers.kaction];
    platforms = lib.platforms.unix;
  };
}
```

Reply:
```
; alejandra --version
Alejandra 1.2.0
```

Reply:
alejandra is incompatible with the formatting used in nixpkgs. Please don't use it.